test/specs/behaviors/testDefinitions.ts

```diff
@@ -162,6 +162,27 @@ definitions.push({
   },
 })
 
+// Example:  Adds attribute 'aria-expanded=true' based on the property 'open' and 'hasSubtree'
+definitions.push({
+  regexp: /Adds attribute '([\w\-\w \s*]+)=([a-z]+)' based on the property '([a-z]+)' and '([a-zA-Z]+)'/g,
+  testMethod: (parameters: TestMethod) => {
+    const [
+      attributeToBeAdded,
+      attributeExpectedValue,
+      propertyDependingOnFirst,
+      propertyDependingOnSecond,
+    ] = [...parameters.props]
+    const property = {}
+    property[propertyDependingOnFirst] = attributeExpectedValue
+    property[propertyDependingOnSecond] = true
+
+    const expectedResult = parameters.behavior(property).attributes.root[attributeToBeAdded]
+    expect(testHelper.convertToBooleanIfApplicable(expectedResult)).toEqual(
+      testHelper.convertToBooleanIfApplicable(attributeExpectedValue),
```

Comment:
This is good test 👍 

but i think we could cover more path of the expression which we have in behavior, because with this test we cover only 1/3 of the statement. 

I was trying myself and we could extend like to more statement to be tested:
  const propertySecondPropUndefined = {}
    propertySecondPropUndefined[propertyDependingOnFirst] = attributeExpectedValue
    propertySecondPropUndefined[propertyDependingOnSecond] = undefined
    const expectedResultSecondPropUndefined = parameters.behavior(propertySecondPropUndefined).attributes.root[attributeToBeAdded]
    expect(testHelper.convertToBooleanIfApplicable(expectedResultSecondPropUndefined)).toEqual(
      testHelper.convertToBooleanIfApplicable(undefined),
    )

    const propertyFirstPropNegate = {}
    propertyFirstPropNegate[propertyDependingOnFirst] = !attributeExpectedValue
    propertyFirstPropNegate[propertyDependingOnSecond] = true
    const expectedResultFirstPropertyNegate = parameters.behavior(propertyFirstPropNegate).attributes.root[attributeToBeAdded]
    expect(testHelper.convertToBooleanIfApplicable(expectedResultFirstPropertyNegate)).toEqual(
      testHelper.convertToBooleanIfApplicable(!attributeExpectedValue),
    )